lib/3scale_toolbox/commands/remote_command.rb

```diff
@@ -0,0 +1,61 @@
+require 'cri'
+require '3scale_toolbox/base_command'
+require '3scale_toolbox/commands/remote_command/remote_add'
+require '3scale_toolbox/commands/remote_command/remote_remove'
+require '3scale_toolbox/commands/remote_command/remote_rename'
+
+module ThreeScaleToolbox
+  module Commands
+    module RemoteCommand
+      class RemoteCommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+        def self.command
+          Cri::Command.define do
+            name        'remote'
+            usage       'remote <command> [options]'
+            summary     '3scale CLI remote'
+            description '3scale CLI command to manage your remotes'
+            runner RemoteCommand
+          end
+        end
+
+        def run
+          list_remotes
+        rescue StandardError => e
```

Comment:
Do you have RuboCop/CodeClimate set up? 

There is no need for explicitly catch `StandardError` as `rescue => e` does the same and requires less thinking for the reader to figure out what it actually does.

Reply:
RuboCop is my friend

```bash
$ rubocop -v
0.59.2
```
Rubocop is not warning, with no specific conf, all settings are default.

Anyway, error handling will be done globally and not by command, so that code is being removed.